readme.md

```diff
@@ -261,6 +261,26 @@ Middleware relating to CORS will cause the app to report Unknown Error when logg
 From [@Dondochaka](https://discord.com/channels/942908292873723984/942914154254176257/945074590374318170) and [@BeastleeUK](https://discord.com/channels/942908292873723984/942914154254176257/970366039294611506)
 <br />
 
+### Example Caddyfile - [Caddy Reverse Proxy](https://caddyserver.com/docs/caddyfile/directives/reverse_proxy)
+
+```
+subdomain.domain.com {
+        tls internal
+        header {
+                Strict-Transport-Security max-age=31536000
+                X-Content-Type-Options nosniff
+                X-Frame-Options DENY
+                Referrer-Policy no-referrer-when-downgrade
+                X-XSS-Protection 1
+        }
+        encode gzip zstd
+        reverse_proxy LOCAL_IP:PORT {
+                trusted_proxies xxx.xx.xxx.x/xx xxx.xx.xxx.x/xx
+        }
+}
```

Comment:
I'm not sure including `tls internal` in the example is a good idea. Also, I'm not sure if those headers are necessary.
The `trusted_proxies` option of `reverse_proxy` is only necessary if you are using Cloudflare tunnels or a CDN, I think.

Personally, this is what I do in my Caddyfile:

```
abs.example.com {
	reverse_proxy * <LOCAL_IP>:<PORT>
	encode zstd gzip
}
```

Reply:
> I'm not sure including `tls internal` in the example is a good idea. 

Probably not, that's a situational thing.

> Also, I'm not sure if those headers are necessary.

They are not, just tested on my setup and it worked without them. Not sure why I had them there.

> The trusted_proxies option of reverse_proxy is only necessary if you are using Cloudflare tunnels or a CDN, I think.

You are correct, I use it because I have Cloudflare as my DNS provider.